### PR TITLE
[Merged by Bors] - chore: deduplicate ToExpr FilePath

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

File: Mathlib/Tactic/ToExpr.lean [hunk 1]
```diff
@@ -37,6 +37,11 @@ attribute [-instance] Lean.instToExprProd
 
 deriving instance ToExpr for Prod
 
+open System in
+instance : ToExpr FilePath where
+  toTypeExpr := mkConst ``FilePath
+  toExpr path := mkApp (mkConst ``FilePath.mk) (toExpr path.1)
```
Comment (eric-wieser) [inline Mathlib/Tactic/ToExpr.lean:43]: Would `deriving instance ToExpr for FilePath` have worked here?

Comment (semorrison) [reply Mathlib/Tactic/ToExpr.lean]: Took me a moment to discover that `whatsnew in` was the easiest way to see what `deriving instance` does here, but yes, it is correct. I've changed it out.

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**